src/seedu/addressbook/commands/Command.java

```diff
@@ -37,14 +37,10 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
     }
 
     /**
-     * Executes the command and returns the result.
+     * Ensure that all commands implement an execute function.
```

Comment:
If your child-class doesn't override the header comment, it'll see this for the header comment, which would look strange. I would thus think that keeping the old header as it is is alright.